### PR TITLE
remove unused files, improve project structure

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

File: docs/Additional Information/hackathon-rules-waiver-release-and-code-of-conduct.md [hunk 1]
```diff
@@ -1,98 +0,0 @@
----
-layout: nodes.liquid
```
Comment (StephenFluin) [inline docs/Additional Information/hackathon-rules-waiver-release-and-code-of-conduct.md:2]: Don't we need this for our future hackathons and events? @PatrickAlphaC @pappas999

File: docs/Contract Creators/addresses-and-job-ids.md [hunk 2]
```diff
@@ -1,106 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Contract Creators/addresses-and-job-ids.md:1]: Add redirects for pages like this one.

File: docs/Contract Creators/contract-creators-overview.md [hunk 3]
```diff
@@ -1,45 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Contract Creators/contract-creators-overview.md:1]: Redirect this one.

File: docs/Contract Creators/on-chain-warning-flag.md [hunk 4]
```diff
@@ -1,73 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Contract Creators/on-chain-warning-flag.md:1]: Keep this

File: docs/Contract Creators/using-chainlink-price-reference-contracts.md [hunk 5]
```diff
@@ -1,509 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Contract Creators/using-chainlink-price-reference-contracts.md:1]: redirect

File: docs/Developer Reference/user-guides/paying-for-smart-contracts-oracles.md [hunk 6]
```diff
@@ -1,92 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Developer Reference/user-guides/paying-for-smart-contracts-oracles.md:1]: Redirect

File: docs/Node Operators/adding-jobs-to-your-node.md [hunk 7]
```diff
@@ -1,67 +0,0 @@
----
```
Comment (StephenFluin) [inline docs/Node Operators/adding-jobs-to-your-node.md:1]: redirect

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: docs/Node Operators/adding-jobs-to-your-node.md (hunk 7 above)
Comment (bencxr) [reply docs/Node Operators/adding-jobs-to-your-node.md:1]: does this exist somewhere else now?

File: docs/Contract Creators/using-chainlink-price-reference-contracts.md (hunk 5 above)
Comment (bencxr) [reply docs/Contract Creators/using-chainlink-price-reference-contracts.md:1]: this exists somewhere else now?

Comment (dmitrydao) [reply docs/Contract Creators/using-chainlink-price-reference-contracts.md:1]: The price feeds are on the [LINK Token Contracts](https://docs.chain.link/docs/link-token-contracts/) page and an explanation of how to use them is in the [tutorials](https://docs.chain.link/docs/beginners-tutorial/) section.

File: docs/Node Operators/adding-jobs-to-your-node.md (hunk 7 above)
Comment (dmitrydao) [reply docs/Node Operators/adding-jobs-to-your-node.md:1]: Nope but it is fully replaced by the [LINK Token Contracts](https://docs.chain.link/docs/link-token-contracts/).

**Comment by StephenFluin** [comment]
Once this is rebased on the latest from `main` and the requested changes have been made, I think this is good to merge. A handful of folks have signed off that we don't need the deleted pages (@bencxr @PatrickAlphaC @alexroan @nzleet ).

**Comment by dmitrydao** [comment]
> Once this is rebased on the latest from `main` and the requested changes have been made, I think this is good to merge. A handful of folks have signed off that we don't need the deleted pages (@bencxr @PatrickAlphaC @alexroan @nzleet ).

Ok, I just rebased it. Let me know if it looks good.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
I just did a linkcheck on the test deployment and apparently this breaks the navigation because all of the feeds links break because they were moved. Can you not move them in this PR? We can consider putting them in a folder later.

**Comment by StephenFluin** [comment]
@dmitrydao Please run a linkcheck yourself. Still has errors.

```
https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/lcx-testnet/
- (46:468) 'Multiply' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#multiply (HTTP 404)
- (46:521) 'EthUint256' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethuint256 (HTTP 404)
- (46:578) 'EthTx' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethtx (HTTP 404)
- (46:468) 'Multiply' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#multiply (HTTP 404)
- (46:521) 'EthUint256' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethuint256 (HTTP 404)
- (46:578) 'EthTx' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethtx (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/make-a-http-get-request/
- (77:956) 'adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/miscellaneous/
- (1:8870) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9173) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/node-operator-overview/
- (1:8766) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9087) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/node-operators/
- (1:8952) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9257) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (15:57) 'core ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (1:8952) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9257) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (15:57) 'core ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/off-chain-reporting/
- (3:3991) 'Price Fe..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/migrating-to-flux-aggregator (HTTP 404)
- (3:4276) '› Migrat..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/migrating-to-flux-aggregator/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/performing-system-maintenance/
- (1:9078) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9413) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/run-an-ethereum-client/
- (1:8987) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9308) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/running-a-chainlink-node/
- (1:9083) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9408) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (1:11371) 'Adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (1:9083) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
- (1:9408) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
- (1:11371) 'Adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)

Errors. Checked 13153 links, 192 destination URLs (440 ignored), 8 have errors, 13 have warnings.

```

**Comment by dmitrydao** [comment]
> @dmitrydao Please run a linkcheck yourself. Still has errors.
> 
> ```
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/lcx-testnet/
> - (46:468) 'Multiply' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#multiply (HTTP 404)
> - (46:521) 'EthUint256' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethuint256 (HTTP 404)
> - (46:578) 'EthTx' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethtx (HTTP 404)
> - (46:468) 'Multiply' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#multiply (HTTP 404)
> - (46:521) 'EthUint256' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethuint256 (HTTP 404)
> - (46:578) 'EthTx' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/#ethtx (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/make-a-http-get-request/
> - (77:956) 'adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/miscellaneous/
> - (1:8870) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9173) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/node-operator-overview/
> - (1:8766) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9087) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/node-operators/
> - (1:8952) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9257) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (15:57) 'core ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (1:8952) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9257) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (15:57) 'core ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/off-chain-reporting/
> - (3:3991) 'Price Fe..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/migrating-to-flux-aggregator (HTTP 404)
> - (3:4276) '› Migrat..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/migrating-to-flux-aggregator/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/performing-system-maintenance/
> - (1:9078) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9413) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/run-an-ethereum-client/
> - (1:8987) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9308) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/running-a-chainlink-node/
> - (1:9083) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9408) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (1:11371) 'Adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (1:9083) 'FluxAggr..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/fluxaggregator-uses/ (HTTP 404)
> - (1:9408) 'Core Ada..' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> - (1:11371) 'Adapters' => https://deploy-preview-210--dreamy-villani-0e9e5c.netlify.app/docs/adapters/ (HTTP 404)
> 
> Errors. Checked 13153 links, 192 destination URLs (440 ignored), 8 have errors, 13 have warnings.
> ```

I think it was conflicting with another PR. Should be fixed now.

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**